cmd/gapir/cc/main.cpp

```diff
@@ -285,6 +305,14 @@ struct Options {
     }
     return opts;
   }
+
+ private:
+  void SetMode(ReplayMode mode) {
+    if (this->mode != kUnknown && this->mode != mode) {
+      mode = kConflict;
```

Comment:
Should there be a 'return' after this?

Reply:
`mode` is public, so we don't need to return anything.